app/controllers/ChannelTestsController.scala

```diff
@@ -71,11 +66,15 @@ abstract class LockableS3ObjectController[CT <: ChannelTests[_] : Decoder : Enco
     */
   def get = authAction.async { request =>
     runWithLockStatus { case VersionedS3Data(lockStatus, _) =>
-      S3Json
-        .getFromJson[CT](s3Client)
-        .apply(dataObjectSettings)
-        .map { case VersionedS3Data(value, version) =>
-          Ok(noNulls(LockableS3ObjectResponse(value, version, lockStatus, request.user.email).asJson))
+      dynamo.getAllTests[T](channel)
+        .map { channelTests =>
+          val response = LockableS3ObjectResponse(
+            ChannelTests(channelTests),
+            "versioning-deprecated",
```

Comment:
the client still expects a version field, though this isn't used now that we're not using s3. This can be removed later